### PR TITLE
Import boilerplate

### PR DESCRIPTION
**Pull request by KenwoodFox**
@Kredcool You must review this.

### CODE REVIEW

**Comment by KenwoodFox** [comment]
This is ready to merge.

File: .github/workflows/build-workflow.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        run: |
+          cd rio && timeout 20 pipenv run python robot.py sim --nogui || code=$?; if [[ $code -ne 124 && $code -ne 0 ]]; then exit $code; fi
+      
+      - name: Discord notification
+        if: ${{ failure() }}
+        env:
+          DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
+        uses: Ilshidur/action-discord@master
+        with:
+          args: '@Kode Team Simulator failed in {{ EVENT_PAYLOAD.pull_request.html_url }} !'
```
Comment (Kredcool) [inline .github/workflows/build-workflow.yml]: it should be @Software Team not @Kode Team because that will not ping anyone

File: .gitignore [hunk 2]
```diff
@@ -1,129 +1,20 @@
...
+dashboard/www/main.js*
+#dashboard packages
+node_modules
+_staging
+
+# Tools
+.SysId/
+.OutlineViewer/
+*.out
+sysid_data*
```
Comment (Kredcool) [inline .gitignore:20]: this appears to be only checking for shouldn't it be checking for more?
imgui.ini
.vscode
simgui*
__pycache__
_build
*.*~
.python-version
dashboard/www/main.js*
node_modules
_staging
.SysId/
.OutlineViewer/
*.out
sysid_data*

File: README.md [hunk 3]
```diff
@@ -1 +1,5 @@
-# 1721-ChargedUp
\ No newline at end of file
+[![Robot Workflow](https://github.com/FRC-1721/1721-ChargedUp/actions/workflows/build-workflow.yml/badge.svg)](https://github.com/FRC-1721/1721-ChargedUp/actions/workflows/build-workflow.yml)
+[![Extras Workflow](https://github.com/FRC-1721/1721-ChargedUp/actions/workflows/extras-workflow.yml/badge.svg)](https://github.com/FRC-1721/1721-ChargedUp/actions/workflows/extras-workflow.yml)
+[![Documentation Status](https://readthedocs.org/projects/1721-chargedup/badge/?version=latest)](https://1721-chargedup.readthedocs.io/en/latest/?badge=latest)
+
+# 1721-ChargedUp
```
Comment (Kredcool) [inline README.md:5]: the # 1721-ChargedUp on top before the other things

**Review by Kredcool — APPROVED** [review]
The documentation needs to be updated after we have a functioning robot, and all of the templates code shall most likely be rewritten, but this looks like a decent start.

File: .gitignore (hunk 2 above)
Comment (KenwoodFox) [reply .gitignore:20]: Not sure what you mean, this should be everything?

File: README.md (hunk 3 above)
Comment (KenwoodFox) [reply README.md:5]: Changed to what :P

File: .github/workflows/build-workflow.yml (hunk 1 above)
Comment (KenwoodFox) [reply .github/workflows/build-workflow.yml]: Fixed in 93ce54c

File: .gitignore (hunk 2 above)
Comment (Kredcool) [reply .gitignore:20]: it just appears to be getting rid of the default so, I don't want a bunch of caches to be there.

File: README.md (hunk 3 above)
Comment (Kredcool) [reply README.md:5]: I just check, your doing it right, I'm not used to putting those on README.md.

File: .gitignore (hunk 2 above)
Comment (KenwoodFox) [reply .gitignore:20]: Default is all in there don't worry everything from last year is ignored

File: README.md (hunk 3 above)
Comment (KenwoodFox) [reply README.md:5]: No problem!

**Pull request merged**